server_admin/topics/sso-protocols/oidc.adoc

```diff
@@ -97,6 +97,225 @@ This is used by clients running on internet-connected devices that have limited
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.5[CIBA Specification].
+
+|Expires In
+|The expiration time of the "auth_req_id" in seconds since the authentication request was received. The default setting is 120. This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Interval
+|The interval in seconds the CD (Consumption Device) needs to wait for between polling requests to the token endpoint. The default setting is 5. This configuration is optional.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Authentication Requested User Hint
+|The way of identifying the end-user for whom authentication is being requested. The default setting is "login_hint".  There are three modes, "login_hint", "login_hint_token" and "id_token_hint". {project_name} only supports "login_hint". This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.7.1[CIBA Specification].
+
+|===
+
+====== Provider Setting
+
+The CIBA grant uses the following two providers.
+
+. Authentication Channel Provider : provides the communication between {project_name} and the entity that actually authenticates the user via AD (Authentication Device).
+. User Resolver Provider : get `UserModel` of {project_name} from the information provided by the client to identify the user.
+
+{project_name} has both default providers. However, the administrator needs to set up Authentication Channel Provider like this:
+
+[source,xml]
+----
+<spi name="ciba-auth-channel">
+    <default-provider>ciba-http-auth-channel</default-provider>
+    <provider name="ciba-http-auth-channel" enabled="true">
+        <properties>
+            <property name="httpAuthenticationChannelUri" value="https://backend.internal.example.com/auth"/>
+        </properties>
+    </provider>
+</spi>
+----
+
+The configurable items and their description follow.
+
+|===
+|Configuration|Description
+
+|httpAuthenticationChannelUri
+|Specifying URI of the entity that actually authenticates the user via AD (Authentication Device).
+
+|===
+
+====== Authentication Channel Provider
+
+CIBA standard document does not specify how to authenticate the user by AD. Therefore, it might be implemented at the discretion of products. Keycloak delegates this authentication to an external authentication entity. To communicate with the authentication entity, keycloak provides Authentication Channel Provider.
```

Comment:
Is it possible to replace `Keycloak delegates ...` with something like `{project_name} delegates ...` ? This is needed so that for RHSSO documentation, the generated text will be "RHSSO delegates ... "

There is lots of other occurences of word "keycloak" in the newly added section you added, so it seems that all of those might need to be changed to "{project_name}"

Reply:
Sorry, it's my mistake. I'll fix it.